tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh

```diff
@@ -188,6 +211,18 @@ function test_container()
       die "FAIL: MKL enabled test in ${TEMP_IMAGE_NAME}"
   fi
 
+  # Test to check if horovod is installed successfully
+  if [[ ${ENABLE_HOROVOD} == "yes" ]]; then
+      debug "Test horovod in the container..."
+      HOROVOD_TEST_CMD=$(${DOCKER_BINARY} exec ${CONTAINER_ID} bash -c "${PYTHON} -c 'import horovod.tensorflow as hvd;'")
+      ${HOROVOD_TEST_CMD}
```

Comment:
Can we skip storing the command in the variable? It's not used anywhere else.
```suggestion
      $(${DOCKER_BINARY} exec ${CONTAINER_ID} bash -c "${PYTHON} -c 'import horovod.tensorflow as hvd;'")
```

Also, does the command need to be wrapped in another shell? If not, please remove the `$()` too.